USE-CASES/hazard-annotations.md

```diff
@@ -0,0 +1,135 @@
+
+It could be as coarse as cover any access to the device or specific to the point of selecting a range of operation within the single specific interaction, e.g.:
+- Accessing a connected camera is a privacy hazard
+- Setting an high level of brightness on an halogen lamp might involve risks of overheating it or setting something nearby on fire.
+
+Some devices may provide additional information on hazards they may simply detect and default policies other devices may try to enforce e.g.:
+- A smart meter may notify that the energy flowing through it is over a threshold and the information may trigger a policy that sets some smart switches off.
+
+Some devices may provide information that can be aggregated together to derive/evaluate a risk:
+- A CO₂/VOC sensor may provide a default policy that modulate the smoke and fire hazard evaluation based on its own information, but in itself may not detect smoke or temperature.
+- A connected Oven may not have a smart meter embedded but may provide enough information to desume the energy consumption by the temperature of its heating elements.
+
+### Dependencies - Affected WoT deliverables and/or work items:
+
+The current Thing Description is adequate to support the model, the hazards annotations can be bound to the specific Form Operation for each Interaction Affordance.
+Additional vocabulary terms may be needed to support a finer level of granularity or declare default policies or formulas to derive additional hazard based on the aggregation of information provided by different devices..
+
+### Description:
+
+The user may have a mean of setting policies to restrict interactions with connected devices, e.g.:
+- Stop any interaction with the TV between 3:00PM and 5:00PM if an unattended children is detected.
+- Require stronger authentication to open a door if the user is considered already inside the house.
+
+The home system may automously change the state of a device to reduce the risk level, e.g.:
+- Reduce the brightness of an Halogen Lamp
+- Start the A/C system if a room temperature is outside the bearable range and a person is detected inside.
+
+The same considerations apply to larger scale (e.g. apartment buildings, neighbourhoods, cities) or slightly different contexts such as industry or hospitals.
+
+The risk categories may be diversified to cover field-specific risks, but the structure remains the same:
+- The Thing Description would provide the hazard information
+- Consumer may act on it or enrich/aggregate it and pass it forward for further processing
+
+### Security Considerations:
+
+The system relies on being able to trust the information exchanged and has to detect bad actors.
+
+### Privacy Considerations:
+
+Privacy is one of the possible category of hazards that may be accounted for, it is no different from a physical hazard in this use-case.
+
+### Accessibility Considerations:
+
+The system may be extended to cover accessibility concerns and have specific policies.
```

Comment:
Should mention that alerts of hazards should be accessible, e.g. perceivable by people with different abilities.  For example, an alarm should be both visual and audible.  An alarm device that is only audible (e.g. a bell, but no light) may have an "Inaccessible to people with hearing disorders" "hazard" associated with it.

Reply:
Still think a little more text would be useful here.